easybuild/tools/filetools.py

```diff
@@ -929,25 +929,26 @@ def move_logs(src_logfile, target_logfile):
 
 
 def cleanup(logfile, tempdir, testing):
-    """Cleanup the specified log file and the tmp directory"""
-    if not build_option('cleanup_tmpdir'):
-        print_msg('Keeping temporary log file(s) and directory.', log=None, silent=testing)
-        return
+    """Cleanup the specified log file and the tmp directory, if desired."""
 
-    if not testing and logfile is not None:
-        try:
-            for log in glob.glob('%s*' % logfile):
-                os.remove(log)
-        except OSError, err:
-            raise EasyBuildError("Failed to remove log file(s) %s*: %s", logfile, err)
-        print_msg('temporary log file(s) %s* have been removed.' % (logfile), log=None, silent=testing)
+    if build_option('cleanup_tmpdir') and not testing:
+        if logfile is not None:
+            try:
+                for log in glob.glob('%s*' % logfile):
```

Comment:
okay, be shouldn't we use that specific structure? I'm not sure I'm comfortable with a general wildcard here